index.html

```diff
@@ -60,8 +76,10 @@
   <script type="text/javascript" src="/static/env.js"></script>
   <div id="app">
     <div id="noscript" class="app__init">
-      <div class="app__init-load-spinner"></div>
-      <img class="app__init-logo" src="/static/logo.svg">
+      <img class="app__init-logo-text" src="/static/logo_text.svg">
+      <img class="app__init-left-face-logo" src="/static/left_face_of_logo.svg">
+      <img class="app__init-top-face-logo" src="/static/top_face_of_logo.svg">
+      <img class="app__init-right-face-logo" src="/static/right_face_of_logo.svg">
```

Comment:
wrap logo parts into a div

Reply:
pls rename the .svg files to kebab-case (for the consistency, all the other files will be renamed one day also)

Reply:
ok